config.toml.template

```diff
@@ -0,0 +1,20 @@
+# Uncomment a job and its arguments to start it.
+
+# [cubic_archive_qlik]
+# save_local = true
+# source = "mbta-ctd-dataplatform-dev-springboard/odin/data/cubic_qlik/"
+# archive_processed = "mbta-ctd-dataplatform-dev-archive/odin/archive/cubic_qlik/processed/"
+# # for qlik/utils.py and qlik/clean.py
+# source_prefixes = [
+#   "mbta-ctd-dataplatform-dev-archive/cubic/ods_qlik/",
+#   "mbta-ctd-dataplatform-dev-error/cubic/ods_qlik/"
+# ]
+# ds_prefix = "mbta-ctd-dataplatform-dev-springboard/odin/data/cubic_qlik/"
+# # for qlik/clean.py
+# archive_ignored = "mbta-ctd-dataplatform-dev-archive/odin/archive/cubic_qlik/ignored/"
+# archive_error = "mbta-ctd-dataplatform-dev-archive/odin/error/cubic_qlik/"
```

Comment:
These names probably aren't very good because I don't understand the file organization well enough to pick better names. If you have suggestions on what to name these paths I'll change them.